install.sh

```diff
@@ -56,11 +64,19 @@ wo.wolocalroot=$wodir
 wo.dir.user.home.library.frameworks=$HOME/Library/Frameworks
 EOF
 sudo chown -R $USER $wodir/Library/Frameworks
+echo Done!
```

Comment:
It's good practice to surround strings with `!` in them with `'` (single quotes), since `!` has some special meaning in shells, and can cause problems; see e.g. https://unix.stackexchange.com/questions/3747/understanding-the-exclamation-mark-in-bash